x-pack/plugins/enterprise_search/public/applications/workplace_search/components/overview/recent_activity.scss

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.activity-feed {
+  td:last-child {
+    text-align: right;
+  }
+}
+
+.activity {
+  &__error {
+    color: #f95e5b !important;
+    background: #ffefeb;
+    border-bottom-color: transparent;
+
+    a,
+    td {
+      color: #f95e5b;
+      font-weight: 600;
```

Comment:
```suggestion
    color: $euiColorDanger !important;
    background: opacity($euiColorDanger, 0.1);
    border-bottom-color: transparent;

    a,
    td {
      color: $euiColorDanger;
      font-weight: 600;
```

Reply:
Just a minor design update here to align better with Elastic brand colors for error states.

Reply:
Addressed in 4bd9745